sdk/src/client.rs

```diff
@@ -44,6 +44,9 @@ pub trait SyncClient {
     /// Get account balance or 0 if not found.
     fn get_balance(&self, pubkey: &Pubkey) -> Result<u64>;
 
+    /// Get account balance or 0 if not found. Does not wait for default confirmations.
+    fn get_balance_now(&self, pubkey: &Pubkey) -> Result<u64>;
```

Comment:
I sorta just want replace the `_now()` variants with `_with_confidence()`, then add a constant that allows us to say `_with_confidence(Confidence::recent)` or `_with_confidence(Confidence::default)`, etc

Reply:
I see what you mean conceptually, but what does that constant look like in practice?

Reply:
`RpcConfidenceConfig`?

Reply:
Oh. Stash that in sdk, then?

Reply:
Sure, sans the Rpc prefix or something like that